build-system/pr-check/checks.js

```diff
@@ -42,8 +42,8 @@ function pushBuildWorkflow() {
   timedExecOrDie('amp check-exact-versions');
   timedExecOrDie('amp check-renovate-config');
   timedExecOrDie('amp server-tests');
-  timedExecOrDie('amp make-extension --name=t --test --cleanup');
-  timedExecOrDie('amp make-extension --name=t --test --cleanup --bento');
+  timedExecOrDie('amp make-extension --name=foo --test --cleanup');
+  timedExecOrDie('amp make-extension --name=foo --test --cleanup --bento');
```

Comment:
To further simplify things, I wonder if `--test` should automatically imply `--name=foo` and `--cleanup`. With this, callers like `pr-check/checks.js` don't have to know about the internal implementation of `amp make-extension`, and can merely call it with `--test` and `--test --bento`. WDYT?

Reply:
That's a good point. I'll defer to @alanorozco who set the checks up, since I've since modified the PR not to mak this change -- Thanks for the quick look though! 

Anecdotally, the `amp make-extension --test` command has been helpful for me to debug with a variety of `--name` values, and removing `--cleanup` has been necessary to preserve the files so I may dig into them when the template tests fail. But we could have smart, over-ridable defaults like you suggest to cover most use cases, as I imagine CI runs this command the most by far.

Reply:
`--test` maybe shouldn't imply `--cleanup`. consider the case where you want to generate an extension, test the generated code to ensure you're working from a functional base, then iterate. You don't want the cleanup

Reply:
Both use cases can be satisfied by applying the defaults only when `isCiBuild()` is `true`.